Makefile.in

```diff
@@ -32,12 +36,18 @@ SRC_FILES := $(wildcard src/*.cpp) \
 	     $(wildcard src/bvals/cc/mg/*.cpp) \
 	     $(wildcard src/bvals/fc/*.cpp) \
 	     $(wildcard src/bvals/orbital/*.cpp) \
+	     $(wildcard src/bvals/sixray/*.cpp) \
 	     $(wildcard src/bvals/utils/*.cpp) \
 	     $(wildcard src/coordinates/*.cpp) \
 	     src/eos/general/$(GENERAL_EOS_FILE) \
 	     src/eos/$(EOS_FILE) \
 	     src/eos/eos_high_order.cpp \
 	     src/eos/eos_scalars.cpp \
+		 $(wildcard $(CHEMISTRY_FILE)) \
```

Comment:
is this supposed to have a whitespace offset?

Reply:
No. Somehow my editor didn't show that - maybe it's a tab thing. Anyway, now fixed.